tests/installation/resolve_dependency_issues.pm

```diff
@@ -34,6 +34,11 @@ sub run {
 sub post_fail_hook {
     my $self = shift;
     select_console 'root-console';
+    if (check_var('UPGRADE', '1')) {
```

Comment:
I think use UPGRADE as condition is not good, since this also can happen during fresh installation case.
We can check zypper command return code if needed.
Also this logic can be hide into upload_solvertestcase_logs in my view

Reply:
changed